brave/brave-core
# feat(wallet): add squid to cross-chain swap providers

Resolves https://github.com/brave/brave-browser/issues/41049



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open Swap / Bridge
2. Select Squid as the provider
3. Select source and destination tokens
4. Enter a "from" amount

User should be able to see a quote from Squid Router, create a transaction, and confirm it on the blockchain.
